src/devices/Ft4222/Ft4222Gpio.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Ft4222;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Device.Gpio
```

Comment:
Let's change the namespace and have it be Iot.Device.Ft4222.Gpio instead. We shouldn't have device bindings that are putting stuff into the System namespace.

Reply:
@joperezr I'd avoid having device name in the namespace - that is usually causing that it's very annoying to use because you need to write name explicitly. What about `Iot.Device.Gpio` or `Iot.Device.Usb`?

Reply:
I agree it would be a bit better if we had a different namespace naming scheme, but this is what we already do on all of the existing bindings, so at least for now I would aim for consistency. We can log an issue and discuss about this and see if we want to take in the breaking change and re-organize into different namespaces aside from this PR.

Reply:
So which namespace should I use then?